lib/assessment.js

```diff
@@ -39,7 +60,12 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // The function you write should return
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
+ // I am finding this question hard to break down into smaller steps to start this work. I know I need to fun an if function and I know I need a return but not sure what else.
```

Comment:
1. Determine whether an object was passed in. Create one if not.
2. Assign the passed-in `method` to the object's property of name `propertyName`
3. Return the object